db/migrate/20160829080136_add_organization_id_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddOrganizationIdToUsers < ActiveRecord::Migration[5.0]
```

Comment:
@RushilBasappa Merge both the migrations into a single file. You can name it as `AddOriganisationIdToUsersAndTrackTemplates`.